packages/styled/src/base.js

```diff
@@ -196,12 +197,36 @@ let createStyled: CreateStyled = (tag: any, options?: StyledOptions) => {
       nextTag: ElementType,
       nextOptions?: StyledOptions
     ) => {
-      return createStyled(
-        nextTag,
-        nextOptions !== undefined
-          ? { ...(options || {}), ...nextOptions }
-          : options
-      )(...styles)
+      const { label, ...prevOptions } = options || {}
+      let nextStyles = styles
+
+      if (
+        identifierName !== undefined &&
+        (nextOptions && nextOptions.label !== undefined)
+      ) {
+        nextStyles = styles.slice(1)
+
+        if (
+          nextTag.__emotion_real === nextTag &&
+          nextTag.__emotion_identifier !== undefined
+        ) {
+          nextTag = { ...nextTag }
+          nextTag.__emotion_real = nextTag
+          nextTag.__emotion_styles = nextTag.__emotion_styles.slice(1)
+        }
```

Comment:
If we don't remove the prepended label for `nextTag` we end up with duplicate labels.